src/lib/api/ill/serializers.js

```diff
@@ -6,6 +6,16 @@ const LibrarySerializers = {
   },
 };
 
+const BorrowingRequestSerializers = {
+  responseSerializer: function(hit) {
+    return recordResponseSerializer(hit);
+  },
```

Comment:
I am not sure why we wrap it several times. What about the following suggestion, which will also allow us to pass more than one [arguments](https://github.com/inveniosoftware/react-invenio-app-ils/blob/df478ab328e35afc79e3df44180bdf12fb3c4545/src/lib/api/utils.js#L88). 
```suggestion
  responseSerializer: recordResponseSerializer
```